frontend/ui/src/app/tutorials/branding/branding-tutorial.component.html

```diff
@@ -0,0 +1,216 @@
+                </a>
+                <button
+                  (click)="continueFromWelcome()"
+                  type="submit"
+                  class="flex w-full items-center justify-center rounded-lg border border-primary-700 bg-primary-700 px-5 py-2.5 text-sm font-medium text-white hover:border-primary-800 hover:bg-primary-800 focus:outline-none focus:ring-4 focus:ring-primary-300 dark:border-primary-700 dark:bg-primary-600 dark:hover:border-primary-700 dark:hover:bg-primary-700 dark:focus:ring-primary-800 sm:mt-0 sm:w-auto">
+                  Let's Go
+                  <fa-icon class="ml-2" [icon]="faArrowRight"></fa-icon>
+                </button>
+              </div>
+            </div>
+          </cdk-step>
+          <cdk-step label="Branding Options" [stepControl]="brandingFormGroup">
+            <div class="p-8 w-full">
+              <form [formGroup]="brandingFormGroup" (ngSubmit)="continueFromBranding()">
+                <ol
+                  class="ml-4 relative text-gray-900 dark:text-white mb-12 border-s border-gray-200 dark:border-gray-700">
+                  <li class="mb-10 ms-8">
+                    <span
+                      class="absolute flex items-center justify-center w-8 h-8 bg-gray-100 rounded-full -start-4 ring-4 ring-white dark:ring-gray-900 dark:bg-gray-700">
+                      <fa-icon [icon]="faLightbulb"></fa-icon>
+                    </span>
+                    <h3 class="font-medium leading-tight">Branding Options</h3>
+                    <p class="text-sm mt-2">
+                      Imagine you are one of your customers: What would you like to see in the Distr Customer Portal?
+                    </p>
+                  </li>
+                  <li class="mb-10 ms-8">
+                    <span class="absolute flex items-center justify-center -start-2">
+                      <input type="checkbox" [formControl]="brandingFormGroup.controls.titleDone" />
+                    </span>
+                    <h3 class="font-medium leading-tight">Set Customer Portal Title</h3>
+                    <p class="mt-1 text-xs">The title will be shown in the header next to the logo.</p>
+                    <div class="mt-2">
+                      <input
+                        [formControl]="brandingFormGroup.controls.title"
+                        autotrim
+                        type="text"
+                        name="title"
+                        id="title"
+                        class="bg-gray-50 border border-gray-300 text-sm text-gray-900 rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500" />
+                      @if (brandingFormGroup.controls.title.touched && brandingFormGroup.controls.title.errors) {
+                        <div class="mt-2 text-sm text-red-500">Please enter this field.</div>
```

Comment:
Even if the required field blocks the stepper to continue to the next step, the next button is not displayed as disabled (I also worked with the stepper before and couldn't fix it. but in this case we should investigate and check if we can disable to next button, if the input is not valid)